styles.css

```diff
@@ -491,178 +488,178 @@ hr {
-  padding-bottom: 15px;
- }
-
-.me h2 {
-  padding-bottom: 15px;
- }
-
-.tool-list {
-  display: flex;
-  flex-direction: row;
-  width: 100%;
-  padding-top: 38px;
- }
-
-.tool-list > li {
-  height: 122px;
-  flex-direction: column;
-  align-items: flex-start;
-  justify-content: center;
-  margin: 0 1em 1em 0;
-  width: 120px;
- }
-
-.skills {
-  padding-top: 20px;
- }
-
-.contact-form p {
-  width: 47%;
- }
-
-.respo-icons {
-  display: none;
- }
-
-.frameworks {
-  padding-top: 20px;
- }
-
-.ul-form {
-  list-style: none;
- }
-
-#name_form {
-  padding-left: 20px;
- }
-
-#email_form {
-  padding-left: 20px;
- }
-
-#message {
-  padding-left: 20px;
- }
-
-.main-icons {
-  display: block;
-  width: 55%;
-  height: auto;
-  margin: 10px 0 10px 10px;
- }
+  .container1 {
+    height: 916px;
+  }
+
+  nav {
+    height: 72px;
+  }
+
+  .logo {
+    margin-left: 2em;
+  }
+
+  .mobile-icons {
+    display: none;
+  }
+
+  .home-icons {
+    display: flex;
+  }
+
+  .P-Text {
+    align-content: center;
+    width: 63%;
+  }
+
+  .P-Text h1 {
+    font-size: 48px;
+  }
+
+  .P-Text p {
+    font-size: 20px;
+  }
+
+  .page-2-description {
+    width: 100%;
+  }
+
+  .project1 {
+    flex-direction: row;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
+  .project2 {
+    flex-direction: row-reverse;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
+  .project3 {
+    flex-direction: row;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
+  .project4 {
+    flex-direction: row-reverse;
+    column-gap: 1rem;
+    width: 90%;
+  }
+
+  .web-icons {
+    width: 394px;
+  }
+
+  .pager-head {
+    font-size: 38px;
+    line-height: 52px;
+  }
+
+  .program-used {
+    padding: 20px;
+  }
+
+  .ul-details {
+    font-size: 12px;
+  }
+
+  .p-details {
+    font-size: 18px;
+  }
+
+  .p-ul-details {
+    width: 80%;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .about {
+    flex-direction: row;
+    height: 628px;
+    justify-content: center;
+    align-items: center;
+    padding: 0 8em 0 8em;
+    column-gap: 2em;
+  }
+
+  .user-tool {
+    width: 60%;
+  }
+
+  .user-tool-d {
+    width: 60%;
+    padding-top: 20px;
+    margin-bottom: 20px;
+  }
+
+  .me {
+    width: 70%;
+  }
+
+  .me p {
+    width: 447px;
+    padding-bottom: 15px;
+  }
+
+  .me h2 {
+    padding-bottom: 15px;
+  }
+
+  .tool-list {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    padding-top: 38px;
+  }
+
+  .tool-list > li {
+    height: 122px;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    margin: 0 1em 1em 0;
+    width: 120px;
+  }
+
+  .skills {
+    padding-top: 20px;
+  }
+
+  .contact-form p {
+    width: 47%;
+  }
+
+  .respo-icons {
+    display: none;
+  }
+
+  .frameworks {
+    padding-top: 20px;
+  }
+
+  .ul-form {
+    list-style: none;
+  }
+
+  #name_form {
+    padding-left: 20px;
+  }
+
+  #email_form {
+    padding-left: 20px;
+  }
+
+  #message {
+    padding-left: 20px;
+  }
```

Comment:
The button of the contact form is centered but yours is aligned to the left of the form as shown below. Kindly check and fix it to match the design. :wink: 


### Your design 


![Screenshot from 2023-02-25 17-10-23](https://user-images.githubusercontent.com/65233860/221370192-e803e75b-e54a-4480-8f0c-1a49d43255ae.png)



### Figma design 


![Screenshot from 2023-02-25 16-31-17](https://user-images.githubusercontent.com/65233860/221370045-69195e29-eef9-42a3-90d1-e8a8ce9140f5.png)